frontend/doc-site/01__cellxgene.mdx

```diff
@@ -28,9 +28,9 @@ Main Tasks: <NextLink href="/docs/04__Analyze%20Public%20Data/4_2__scExpression%
 
 A PyPI package to explore and annotate single cell datasets you upload into the tool and explore unpublished single cell data.
 
-Main Tasks: <NextLink href="/docs/05__Annotate%20and%20Analyze%20Your%20Data/5_0__Get%20Started">Analyze & Annotate Private Data</NextLink>, Share & Analyze Private Datasets
+Main Tasks: <NextLink href="/docs/05__Annotate%20and%20Analyze%20Your%20Data/5_0__Get%20Started">Analyze & Annotate Private Data</NextLink>, [Share & Analyze Private Datasets](doc-site/05__Annotate%20and%20Analyze%20Your%20Data)
```

Comment:
If we want the user to be navigated to the link rather than open the link in a new window, we should use `<NextLink/>`